pkg/sidecar/fanout/fanout_handler.go

```diff
@@ -120,5 +123,9 @@ func (f *Handler) dispatch(msg *provisioners.Message) error {
 // makeFanoutRequest sends the request to exactly one subscription. It handles both the `call` and
 // the `sink` portions of the subscription.
 func (f *Handler) makeFanoutRequest(m provisioners.Message, sub eventingduck.ChannelSubscriberSpec) error {
+	if f.config.AsyncHandler {
+		go f.dispatcher.DispatchMessage(&m, sub.SubscriberURI, sub.ReplyURI, provisioners.DispatchDefaults{})
```

Comment:
we will drop all errors with this. I think you could wrap in an anon function to log the error at least:

```
go func () {
  if err := f.dispatcher....; err != nil {
    log err
  } 
  
}
```